src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -28,6 +28,7 @@
     <DevelopmentDependency>true</DevelopmentDependency>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
     <UsePublicApiAnalyzer>false</UsePublicApiAnalyzer>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
```

Comment:
You can set this in the common.project.props, you don't need it in the individual projects.

Reply:
Thanks~ Moved to common.targets, since I found icon is [here](https://github.com/NuGet/NuGet.Client/blob/dev/build/common.targets#L64-L66)